dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/interceptor/JAXRSOutInterceptor.java

```diff
@@ -446,10 +447,10 @@ private void setResponseDate(MultivaluedMap<String, Object> headers, boolean fir
         if (!firstTry || headers.containsKey(HttpHeaders.DATE)) {
             return;
         }
-// Liberty Change for CXF Begin
-//        SimpleDateFormat format = HttpUtils.getHttpDateFormat();
-        headers.putSingle(HttpHeaders.DATE, CachedTime.getCachedTime().getTimeAsString(System.currentTimeMillis()));
-//Liberty Change for CXF End
+		// Liberty Change Start - Pull from chached time
+		// SimpleDateFormat format = HttpUtils.getHttpDateFormat();
+        headers.putSingle(HttpHeaders.DATE, CachedTime.getCachedTime().getTimeAsString(-1));
+		//Liberty Change End
```

Comment:
Curious:  Why was this changed to -1?

Reply:
Yeah this should be undone just like the CachedTime change should be undone.  This is completely undoing my last PR on this.